etc/tutorial/cylc-forecasting-suite/bin/get-observations

```diff
@@ -72,10 +83,22 @@ def get_archived_data(site_id, time):
 
 def extract_observations(data):
     """Extract the relevant observations from the weather data."""
+    try:
+        direction = ORDINATES[data['Period']['Rep']['D']]
+    except KeyError:
+        direction = '0'
+        print 'No wind direction in file so using 0'
+
+    try:
+        speed = data['Period']['Rep']['S']
+    except KeyError:
+        speed = '0'
```

Comment:
I can't remember where type handling gets handled in these scripts, should this be `0` rather than `'0'`?

Reply:
This fix was provided by Joe Hickson, he said
> I know setting speed / direction to a string of ‘0’ (string? Really? Eurgh) is complete lies and not how to handle an absent data point but I am assuming that it will suffice for the tutorial!